ciphers/mixed_keyword_cypher.py

```diff
@@ -1,3 +1,12 @@
+import string
+
+"""This function takes two parameters: a string variable named
+    'key' (with a default value of 'college') and a string variable
+     named 'pt' (with a default value of 'UNIVERSITY').
+     The function returns a string that is a transformation of the 'pt' argument
+      based on a key-shift cipher"""
```

Comment:
Firstly, this isn't inside of a function and isn't formatted
Secondly, what does this tell the user that we don't already know? Everything you said is clearly shown by typehints and naming conventions